wheatley/main.py

```diff
@@ -155,6 +156,15 @@ def main():
     the bot's mainloop.
     """
 
+    # Try to read the file with the version number, if not print an error and set a poison value
+    # as the version
+    try:
+        version_file_path = os.path.join(os.path.split(__file__)[0], "version.txt")
+
+        __version__ = open(version_file_path).read()
```

Comment:
Should this still use a with block to release the file?

Reply:
I thought that happens automatically if you call `read()` on a file object (though quite how that would work I don't know. But I agree it's safer to make it a `with` statement.

Reply:
Done!